snuba/query/processors/tagsmap.py

```diff
@@ -22,6 +26,11 @@ class OptimizableCondition(NamedTuple):
     value: str
 
 
+logger = logging.getLogger("snuba.query-processing")
```

Comment:
Stylistic nit (also, maybe we should start a style guide with some of these decisions, similar to the `ClickHouse` discussion from #641): can we switch to `logging.getLogger(__name__)`?